README.md

```diff
@@ -20,22 +21,33 @@ Telegram local messages anti-revoke plugin.
 
 ## :tomato: Compatibility
 Usually the plugin is not affected by small updates of Telegram.  
-But some updates may cause the plugin to error or crash, please [report the issue](https://github.com/SpriteOvO/Telegram-Anti-Revoke/issues) and wait for the plugin to be updated.
+But some updates may cause the plugin to error or crash, please [report the issue](/issues) and wait for the plugin to be updated.
 
 If you are using **Telegram beta**, please check the latest preview version.  
 (Compatibility updates for **Telegram beta** may not be released in time)
 
 ## :hamburger: Usage
-1. Go to the [Release Page](https://github.com/SpriteOvO/Telegram-Anti-Revoke/releases) to download the latest version of the plugin.  
-2. Close the running Telegram process.  
-3. Copy *version.dll* file into the Telegram root directory.  
-4. Run the Telegram program.
+1. Go to the [Release Page](/releases) to download the latest version of the plugin.  
+2. Open Telegram and open the top-left menu. From there, note the application version and architecture \***ARCH**\* (x86 or x64) which is labelled under its version, e.g "**Version** x.x.x **x64**".
```

Comment:
If the user is using Telegram x86, only "Version x.x.x" will be displayed there.